src/components/gltf-model-plus.js

```diff
@@ -305,6 +305,10 @@ export async function loadGLTF(src, contentType, preferredTechnique, onProgress)
     }
   }
 
+  if (jsonPreprocessor) {
+    parser.json = jsonPreprocessor(parser.json);
```

Comment:
Don't think it matters too much for our current set of preprocessing but it probably makes sense to do this before our other modifications.